tests/generators/test_flatdata_generator.py

```diff
@@ -0,0 +1,142 @@
+'''
+ Copyright (c) 2017 HERE Europe B.V.
+ See the LICENSE file in the root of this project for license details.
+'''
+
+from generator.generators.FlatdataGenerator import FlatdataGenerator
+from generator.tree.builder import SyntaxTreeBuilder
+
+from nose.tools import *
+
+def generate_and_compare(definition, generator, expectation):
+    tree = SyntaxTreeBuilder.build(definition=definition)
+    contents = generator().render(tree)
+    assert_equal.__self__.maxDiff = None
+    assert_equal(expectation, contents);
+
+
+def test_normalization():
+    expected_lines = """namespace ns {
+const i32 D = -10;
+}
+
+namespace ns {
+const u32 C = 268435455;
+}
+
+namespace ns {
+struct S1
+{
+    f0 : u64 : 64;
+}
+}
+
+namespace ns {
+@bound_implicitly( b : .ns.A0.v0, .ns.A0.v1 )
+archive A0
+{
+    v0 : vector< .ns.S1 >;
+    v1 : multivector< 14, .ns.S1 >;
+}
+}
+
+namespace ns {
+struct S0
+{
+    f0 : u64 : 64;
+    f1 : u64 : 64;
+}
+}
+
+namespace ns {
+enum Enum1 : u16
+{
+    A = 1 ,
+    B = 13 ,
+    C = 14 
+}
+}
+
+namespace ns {
+struct XXX
+{
+    e : .ns.Enum1 : 16;
+    f : .ns.Enum1 : 4;
+}
+}
+
+namespace ns {
+@bound_implicitly( foo : .ns.A1.v0, .ns.A1.v2, .ns.A0.v0 )
```

Comment:
Could you please add another bound just to make sure that multiple of them are fine?

Reply:
Done